src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
@jacebrowning so can you change this to accept a single, specific file, and not performing further searches? If you lost interest because of #237, that's ok :)